qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java

```diff
@@ -58,7 +59,9 @@ public void passesCorrectFilesWithNoExceptions() throws Exception {
     /**
      * FindbugsValidator can report incorrectly added throws.
      * @throws Exception If something wrong happens inside
+     * @todo #706 Temporary disable BED_BOGUS_EXCEPTION_DECLARATION check
```

Comment:
@dskalenko this link is pretty old and very generic. Just imagine it is a task with thorough description and time budget